run_linux.go

```diff
@@ -685,6 +697,38 @@ func (b *Builder) generateHosts(rdir, hostname string, addHosts []string, chownO
 	return cfile, nil
 }
 
+// generateHostname creates a containers /etc/hostname file
+func (b *Builder) generateHostname(rdir, hostname string, chownOpts *idtools.IDPair) (string, error) {
+	hostnamePath := "/etc/hostname"
+	stat, err := os.Stat(hostnamePath)
+	if err != nil {
+		return "", err
+	}
```

Comment:
This will fail if /etc/hostname does not exists for some reason. We should no check this. Just use uid/gid = 0 as default. I doubt that this will ever be owned by another uid.

Reply:
fixed

Reply:
@Luap99 PTAL again.